src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyleHtmlComment.java

```diff
@@ -12,7 +12,13 @@
  * @author Tobias Geyer
  * @version 1.0
  */
```

Comment:
remove header and author/version comments

Reply:
done